2024-06-04 22:02:05,058 - DialogueSystem - ERROR - Request id: , Status code: -1, error code: Unknown, error message: Error type: <class 'requests.exceptions.ConnectionError'>, message: HTTPSConnectionPool(host='dashscope.aliyuncs.com', port=443): Max retries exceeded with url: /api/v1/services/aigc/text-generation/generation (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0cba2fe030>: Failed to resolve 'dashscope.aliyuncs.com' ([Errno -3] Temporary failure in name resolution)"))
2024-06-04 22:02:05,058 - DialogueSystem - INFO - 
2024-06-04 22:02:25,131 - DialogueSystem - ERROR - Request id: , Status code: -1, error code: Unknown, error message: Error type: <class 'requests.exceptions.ConnectionError'>, message: HTTPSConnectionPool(host='dashscope.aliyuncs.com', port=443): Max retries exceeded with url: /api/v1/services/aigc/text-generation/generation (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0cba2fe930>: Failed to resolve 'dashscope.aliyuncs.com' ([Errno -3] Temporary failure in name resolution)"))
2024-06-04 22:02:25,131 - DialogueSystem - ERROR - Request id: , Status code: -1, error code: Unknown, error message: Error type: <class 'requests.exceptions.ConnectionError'>, message: HTTPSConnectionPool(host='dashscope.aliyuncs.com', port=443): Max retries exceeded with url: /api/v1/services/aigc/text-generation/generation (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7f0cba2fe930>: Failed to resolve 'dashscope.aliyuncs.com' ([Errno -3] Temporary failure in name resolution)"))
2024-06-04 22:02:25,133 - DialogueSystem - INFO - 
2024-06-04 22:02:25,133 - DialogueSystem - INFO - 
